src/SignalRServiceExtension/Config/IInternalServiceHubContextStore.cs

```diff
@@ -1,12 +1,17 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
+using System.Threading.Tasks;
 using Microsoft.Azure.SignalR;
+using Microsoft.Azure.SignalR.Management;
 
 namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
 {
     internal interface IInternalServiceHubContextStore : IServiceHubContextStore
     {
         AccessKey[] AccessKeys { get; }
+
+        public dynamic GetAsync(Type THubType, Type TType);
```

Comment:
I'm not quite understanding the necessity for this to be separated from `IServiceHubContextStore.GetAsync(string hubName)`. Is this worth documenting?